pkg/webhook/v1alpha2/applicationconfiguration/helper_test.go

```diff
@@ -1,116 +1,16 @@
 package applicationconfiguration
 
 import (
-	"context"
 	"fmt"
 	"testing"
 
-	"github.com/crossplane/crossplane-runtime/pkg/test"
 	"github.com/stretchr/testify/assert"
 
-	json "github.com/json-iterator/go"
-	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"k8s.io/apimachinery/pkg/runtime"
-	"k8s.io/apimachinery/pkg/types"
 	"k8s.io/apimachinery/pkg/util/intstr"
 
 	"github.com/crossplane/oam-kubernetes-runtime/apis/core/v1alpha2"
-	"github.com/crossplane/oam-kubernetes-runtime/pkg/oam/mock"
 )
 
-func TestCheckComponentVersionEnabled(t *testing.T) {
```

Comment:
why delete this test?

Reply:
It's moved to [here](func TestValidateWorkloadNameForVersioningFn(t *testing.T) {) and refactored.